.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*.{kt,kts}]
+max_line_length = 140
+indent_size = 4
+continuation_indent_size = 4
+insert_final_newline = true
```

Comment:
My goal here was to standarize code style settings for the project, I can remove that or move to a separate PR if needed.